Calculator/Calculator.cpp

```diff
@@ -226,12 +331,9 @@ int main() {
 }
 
 /*
-Notes:  Need help figuring out string formatting so character variable can be inserted in error messages.
-
-        When implementing () handling, negatives relating to () will hang in equation
-        Implement function to call from calculator functions to process negatives - 
-        if operator is followed by negative, call process_negative
-        process_negative replaces negative and number entries with a negative number entry
+Notes:  Currently empty parentheses - () - are replaced with a zero.
+        What is the proper way to handle them? Should an error be thrown? Or is that acceptable?
 
-        Related to that - how does a function call another function? I think I'm having scope issues.
+        Need help figuring out string formatting so character variable can be inserted in error messages

```

Comment:
check out `std::stringstream` maybe? or a good old `printf` or `snprintf` is also acceptable, just because it's a C function doesn't mean it doesn't still work in C++